src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -100,7 +567,7 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren('c')).toEqual([span('c:2')]);
   });
 
-  it('works on roots with animation priority before deferred work', () => {
+  it('handles interleaved lo-pri and hi-pri work', () => {
```

Comment:
All of this would make more sense when we treat them as deadlines instead.
This reverse `<=` is super confusing.